scipy/interpolate/_interpolate.py

```diff
@@ -105,7 +105,8 @@ def lagrange(x, w):
 `CloughTocher2DInterpolator`.
 
 For more details see
-`https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff`
+`https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff
+<https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff>`_
```

Comment:
Im not sure if this fixes the issue but I think this should be on line number 130 instead

Reply:
Yeah, absolutely. Thanks, done.